models/model_choice.py

```diff
@@ -36,13 +36,12 @@
             'encoder_depth': 4,
             'decoder_channels': [256, 128, 64, 32]
         }},
-    'fpn_pretrained': {
-        'fct': smp.FPN, 'params': {
-            'encoder_name': 'resnext50_32x4d',
-        }},
-    'pspnet_pretrained': {
-        'fct': smp.PSPNet, 'params': {
-            'encoder_name': "resnext50_32x4d",
+    'unet_plus_pretrained': {
+        'fct': smp.UnetPlusPlus, 'params': {
+            'encoder_name': 'se_resnext50_32x4d',
+            'encoder_depth': 4,
+            'decoder_channels': [256, 128, 64, 32],
+            'decoder_attention_type': 'scse'
```

Comment:
I believe this if not related to single loss mode. In our current workflow, a separate PR would be recommended. Maybe at least slip a note about this off-topic change.